libs/portals/admin/air-discount-scheme/src/screens/CreateDiscount/CreateDiscount.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react'
+
+import {
+  GridRow,
+  GridColumn,
+  GridContainer,
+  ButtonDeprecated as Button,
+  Input,
+  Stack,
+} from '@island.is/island-ui/core'
+import Modal from './components/Modal/Modal'
+import { useCreateExplicitDiscountCodeMutation } from './CreateDiscount.generated'
+
+const AdminCreateDiscount = () => {
+  const [createExplicitDiscountCode] = useCreateExplicitDiscountCodeMutation()
+
+  const [nationalId, setNationalId] = useState('')
+  const [postalcode, setPostalcode] = useState('')
+  const [comment, setComment] = useState('')
+
+  const [discountCode, setDiscountCode] = useState('')
+
+  const [showModal, setShowModal] = useState(false)
+
+  return (
+    <>
+      <GridContainer>
+        <GridRow>
+          <GridColumn span={['12/12', '12/12', '7/12', '8/12', '9/12']}>
+            {discountCode.length ? (
+              <>
+                <h1>{discountCode}</h1>
+                <p>
+                  Umsýsluviðmótið geymir þessa kóða ekki. Þeir munu birtast í
```

Comment:
Thanks for porting this.
Should we add localization?

Reply:
It's not a priority at the moment for admin systems